install/0000_90_machine-api-operator_04_alertrules.yaml

```diff
@@ -18,25 +18,23 @@ spec:
             severity: critical
           annotations:
             message: "machine {{ $labels.name }} does not have valid node reference"
-
----
-apiVersion: monitoring.coreos.com/v1
-kind: PrometheusRule
-metadata:
-  labels:
-    prometheus: k8s
-    role: alert-rules
-  name: machineapioperator-rules
-  namespace: openshift-machine-api
-spec:
-  groups:
-    - name: general.rules
+    - name: machine-api-operator-metrics-collector-up
+      rules:
+        - alert: MachineAPIOperatorMetricsCollectionFailing
+          expr: |
+             mapi_machine-api-operator_collector_up == 0
+          for: 3m
+          labels:
+            severity: critical
+          annotations:
+            message: "machine api operator metrics collection is failing. Check machine api operator logs for more details."
```

Comment:
We could be super helpful and where we say "check ... logs" turn that into something that can be cut/paste: "oc logs -n openshift-machine-api machine-api-operator", for example.

Reply:
We usually focus on precisely describing what happened as this message is also showing up on SRE pager/slack channel/email. After that they are supposed to have runbooks on how to proceed. Personally I don't see any value in getting alert saying me to check logs.

Reply:
On slack you can see examples of how alerts are being reported by going to one of:
#ops-testplatform (managed by DPTP)
#team-monitoring-alert (managed by our SREs)

Reply:
@paulfantom there i saw alert message just saying what problem has happened. eg:
`Alert: TelemeterDown - stage [FIRING:1]  No targets found for Telemeter in namespace . Telemeter is down.`

Providing exact command to check logs i think is an improvement over not suggesting anything.

Reply:
> usually focus on precisely describing what happened

`machine api operator metrics collection is failing.` this is clearly mentioning that metrics collection is failing